cpp.md

```diff
@@ -274,8 +278,56 @@ typedef data-type name;
 typedef unsigned int distance;  // typedef of int 
 ```
 
-## Vectors
-Vectors are same as dynamic arrays. They will be resized on element inseertions & deletions.
+## Sort one-line
```

Comment:
We want to cover basics only in Cheat Sheets, this is advanced information. You can always create these as posts in OneCompiler

Reply:
@onecompiler-ops which things should i remove from cheatsheet please elaborate...I will make it better. Thank you!